package.json

```diff
@@ -1,29 +1,43 @@
 {
-	"private": true,
```

Comment:
For some reason, `package.json` had tabs not spaces. It seems like this should have been covered by the editorconfig? Not sure how that happened, but I switched from tabs to two spaces like the editorconfig says 😄